src/utils/processOutputInterpreters/trivy-image-output.ts

```diff
@@ -0,0 +1,112 @@
+const LineSplitter = /\r?\n/;
+const dateHeader = /^[-\d]+T[-:.\d]+Z?\s+\033\[\d+m([A-Z]+)\033\[\d+m\s+(.*)$/;
```

Comment:
I see this in a run:
```
INFO Detecting Alpine vulnerabilities...

2021-07-30T21:06:15.491Z
	[34mINFO[0m
	Number of language-specific files: 0


Target: moby/buildkit:v0.8.3 (alpine 3.12.7)
```

I suspect that's because `addData` got called with half a line somewhere.

Reply:
Yes, that would explain in. Not great, but it doesn't get in the way of the info we care about. You could create a separate issue for it.